### ISSUE TITLE
bot message - basic include

### ISSUE THREAD

**Issue by ftdgoodluck** [NEAR-DevHub/race-of-sloths#92]
<details>
<summary> 

### 🎉 This pull request is now a part of the [Race of Sloths](https://github.com/akorchyn/test-bot/pull/2)🎉
Thank you @ftdgoodluck for your contribution!

</summary>
<details>
<summary><strong>More info about the author - @ftdgoodluck</strong></summary>

![sloth_stats](https://race-of-sloths.fly.dev/api/badges/akorchyn)

</details> 

<details><summary><strong>What is the Race of Sloths</strong></summary>
<p>

Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag us inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://github.com/race-of-sloths)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://github.com/race-of-sloths)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://github.com/race-of-sloths) for additional details!

</p>
</details> 

<details>
<summary><strong>Bot commands</strong></summary>

- ➕ **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- 🛑 **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
- ⭐ **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
- 🔕 **Pause us:** `@race-of-sloths pause` to stop me on your repo.
</details>

![Sloth Image](https://pbs.twimg.com/profile_images/1791732697376641025/Iw7pTeyp_400x400.jpg)

</details>

_Comment: This block would change as soon as the PR progresses through the scoring/merging process_

#### Current status: [waiting for scoring, stale, waiting for merge, waiting for finalization, executed, excluded]
>[!IMPORTANT]
_//status == waiting for scoring_
We're waiting for maintainer to score this pull request with `@race-of-sloths score X` command

| Reviewer | Score |
|--------|--------|
| @akorchyn  | 5 |
| @botezatunear  | 5 | 

**Final score: 5**

_Comment: appears only once the PR is finalized_
@ftdgoodluck check out your results on the [Race of Sloths Leaderboard!](https://github.com/race-of-sloths)

**Comment by ftdgoodluck** [comment]
For first time Race participants: 
we change the caption "More info about the author block" => "Welcome to the Race of Sloths"
Inside the block we provide the same picture as we use for participants, but without numbers and additional hints on each parameter.
Example:

1. Earn points in the Race with every contribution
2. Compete with other contributors
3. Contribute on a weekly basis to keep the weekly streak
4. Create a game-changer PR monthly to keep the "High Value (TBD)" streak

**Comment by akorchyn** [comment]
![akorchyn (2)](https://github.com/NEAR-DevHub/devbot/assets/42449190/f1778579-feb3-4fb6-804f-1eb22be010e0)

**Comment by botezatunear** [comment]
The first contribution in race-of-sloth ever: "New Sloth joined the Race! Welcome @ftdgoodluck"
The first contribution in race-of-sloth of the month (candidate for +1 monthly streak): "Do you @ftdgoodluck want to apply for monthly streak? Get 8+ score for a single PR this month and receive boost for race-of-sloth"
The first contribution in race-of-sloth of the week (candidate for +1 weekly streak): "Weekly streak is on the road, smart strategy @ftdgoodluck! Secure your streak with another PR"
The 3rd contribution in race-of-sloth during the month : "Are you @ftdgoodluck going to win race-of-sloth this month? If so, you should speed up! Run, Sloth, Run!"
The 4th contribution in race-of-sloth during the month : "You are true leader @ftdgoodluck! Other sloths hot on your heels, keep moving!"
The 5th contribution in race-of-sloth during the month : "Great job @ftdgoodluck! Usain Bolt jealous of your results!"
The 6th contribution in race-of-sloth during the month : "Ilya is looking for your @ftdgoodluck contacts to meet in person and shake your hand!"
The 7th contribution in race-of-sloth during the month : "We are launching a survey "Who is faster? @ftdgoodluck vs SpaceX"? 
The 8th contribution in race-of-sloth during the month : "@ftdgoodluck your results have exceeded our imagination! Our team is creating new congratulatory messages just because of your results!"

**Comment by botezatunear** [comment]
1) The first contribution in race-of-sloth ever: "Let's get ready to rumble! New Sloth joined the Race! Welcome @ftdgoodluck"
2) The first contribution in race-of-sloth of the month (candidate for +1 monthly streak): "@ftdgoodluck starts monthly race-of-sloth, bon voyage!"
3) The first contribution in race-of-sloth of the week (candidate for +1 weekly streak): "@ftdgoodluck keeps weekly streaks and increases chances to win the race, smart strategy! Secure your streak with another PR"
4) The 2nd contribution in race-of-sloth during the month : "@ftdgoodluck kicks into high gear! Run, Sloth, Run!"
The 3rd contribution in race-of-sloth during the month : "It looks like @ftdgoodluck is going to win race-of-sloth this month! Amazing pace!"
5) The 4th contribution in race-of-sloth during the month : "You are true leader @ftdgoodluck! Other sloths hot on your heels, keep moving!"
6) The 5th contribution in race-of-sloth during the month : "Great job @ftdgoodluck! Usain Bolt jealous of your results!"
7) The 6th contribution in race-of-sloth during the month : "Is anyone going to stop this machine?! King of Sloth shakes your @ftdgoodluck hand!"
8) The 7th contribution in race-of-sloth during the month : "Yellow leader's jersey is yours deservedly, @ftdgoodluck! Hat's off to you"
9) The 8th contribution in race-of-sloth during the month : "@ftdgoodluck your results have exceeded our imagination! Our team is creating new congratulatory messages just because of your results!"
